lib/Constants.ts

```diff
@@ -5,6 +5,8 @@ namespace Msal {
     */
     export class Constants {
         static get errorDescription(): string { return "error_description"; }
+        static get error(): string { return "error"; }
+        static get errorCode(): string { return "error_subcode"; }
```

Comment:
do not take dependency for error_subcode. This is not meant to be used in public API.